helm/funcx_endpoint/templates/endpoint-deployment.yaml

```diff
@@ -21,14 +21,14 @@ spec:
       labels:
         app: {{ .Release.Name }}-endpoint
     spec:
-      serviceAccountName: {{ template "funcx_endpoint.fullname" . }}
+      serviceAccountName: {{ template "globus_compute_endpoint.fullname" . }}
       securityContext:
         fsGroup: 1000
       containers:
       - name: {{ .Release.Name }}-endpoint
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         command: [ "/bin/bash", "-c", "--" ]
-        args: [ "/home/funcx/boot.sh {{ coalesce .Values.endpointName .Release.Name }} {{ .Values.endpointUUID }} {{ .Values.endpointCLIargs }}" ]
+        args: [ "/home/compute/boot.sh {{ .Release.Name }} {{ .Values.endpointUUID }} {{ .Values.endpointCLIargs }}" ]
```

Comment:
Why the removal of the `coalesce`?

Reply:
May have been a merge conflict/copy paste error.  Put the coalesce back.